mkosi.md

```diff
@@ -340,6 +340,15 @@ details see the table below.
   decompressed first. This also means that the `shell`, `boot`, `qemu`
   verbs are not available when this option is used.
 
+`--zstd`
+
+: Compress the resulting image with `zstd`. This only applies to
+  `gpt_ext4`, `gpt_xfs`, `gpt_btrfs`, `gpt_squashfs` and is implied
```

Comment:
The line "is implied for `tar`" can only apply to either xz or zstd, but not both. I guess zstd is in every distro now and Arch even defaults to it for packages now, but I'm not the one to make the call to change that default and I guess for this PR keeping xz as the default might be sensible.

Reply:
@behrmann Yes i see that "-J" https://github.com/systemd/mkosi/blob/4ac0cd84e3cc4161d1bff3fd32ba03766a602da6/mkosi#L2911 I'll keep "-J" by default if args.zstd is empty